src/notation/internal/notationinteraction.cpp

```diff
@@ -287,8 +287,8 @@ void NotationInteraction::checkAndShowMScoreError() const
         message = trc("notation", "Please select a note or rest and retry");
         break;
     case MsError::CANNOT_SPLIT_MEASURE_TOO_SHORT:
-        title = trc("notation", "No note or rest selected");
-        message = trc("notation", "Please select a note or rest and retry");
+        title = trc("notation", "Cannot split measure too short");
+        message = trc("notation", "Please select a longer measure and retry");
```

Comment:
I think a better text would be: 

**This measure is too short to be split**

(and no further message).